app/subscriber/src/components/front-page-gallery/FrontPageGallery.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useContent, useNavigateAndScroll } from 'store/hooks';
+import { IContentModel, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+interface IFrontPageGallery extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Content array.
+   */
+  frontpages: IContentModel[];
+}
+
+export const FrontPageGallery: React.FC<IFrontPageGallery> = ({ frontpages }) => {
+  const [srcUrls, setSrcUrls] = React.useState<any[]>([]);
+  const navigateAndScroll = useNavigateAndScroll();
+  const [, { stream }] = useContent();
+
+  React.useEffect(() => {
+    setSrcUrls([]);
```

Comment:
Why do you do this?  You've already set the initial value on line 15.  It will also result in the state getting rendered twice any time `frontpages, stream` dependencies change.

Reply:
Because it's keeps adding when I change the date filter:
![image](https://github.com/bcgov/tno/assets/26801490/e22b2335-63cd-40fd-a466-5f6272451d22)